<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editcomputer</class>
 <widget class="QDialog" name="Editcomputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>491</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="EditComLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Edit computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabelCom">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="comNameLineEdit">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comTypeComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="wasBuiltLabel">
     <property name="text">
      <string>Was the computer built?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="yesRadioButtonCom">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="noRadioButtonCom">
     <property name="text">
      <string>No</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="builtYearLabel">
     <property name="text">
      <string>Year built</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="comBuiltYearSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1500</number>
     </property>
     <property name="maximum">
      <number>2015</number>
     </property>
     <property name="value">
      <number>2015</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmComEditPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backPushButtonEditCom">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
